dvc/stage/__init__.py

```diff
@@ -229,20 +239,20 @@ def _changed_outs(self):
             status = out.status()
             if status:
                 logger.debug(
-                    "Output '{out}' of '{stage}' changed because it is "
+                    "Output '{out}' of {stage} changed because it is "
                     "'{status}'".format(
-                        out=out, stage=self.relpath, status=status[str(out)]
+                        out=out, stage=self, status=status[str(out)]
                     )
                 )
                 return True
 
         return False
 
-    def _changed_md5(self):
-        if self.changed_md5():
+    def stage_changed(self, warn=False):
```

Comment:
Naming is confusing. Not sure how `Stage.stage_changed` and `Stage.changed` are different by the naming.

Reply:
^ For lack of better naming that works on both kind of stages.

Reply:
Any suggestions?

Reply:
None so far. Might get some ideas later.